e2e/tests/notebook/notebook-create-r.spec.ts

```diff
@@ -94,18 +54,4 @@ describe('Create R kernel notebook', () => {
     await analysisPage.deleteResourceFromTable(rNotebookName, ResourceCard.Notebook);
     await analysisPage.waitForLoad();
   });
-
-  // Helper functions: Load previously saved URL instead clicks thru links to open workspace data page.
-  async function loadWorkspace(page: Page, workspaceName?: string): Promise<string> {
```

Comment:
This is the only change that wasn't strictly a move. This function shared workspaces across test runs.

The two tests using this are now run separately. Additionally, creating workspaces per-test decreases complexity.

Uses of this function have been replaced by `await findOrCreateWorkspace(page, { workspaceName });`